src/chrome-launcher.ts

```diff
@@ -177,10 +177,33 @@ class Launcher {
     return DEFAULT_FLAGS.slice();
   }
 
-  static getInstallations() {
+  static getInstallations(): string[] {
+    // fetching the lsregister for all applications can be time consuming
+    // on MacOS therefor look into default paths first
+    if (getPlatform() === 'darwin') {
```

Comment:
Ah, I'm sorry for the miscommunication. I meant `getFirstInstallation` should live near the chrome finder logic that lists all of these darwin priorities. We should definitely leave `getInstallations` alone for this PR.

Sorry for my poor choice of words!

down below in launch we just want to use `getFirstInstallation()` *instead* of `getInstallations()[0]`